src/main/resources/crafter/engine/server-config.properties

```diff
@@ -178,6 +179,15 @@ crafter.security.accessDenied.error.url=/crafter-controller/error/403
 # The XPath queries to extract the roles authorized to access a page
 crafter.security.authorization.page.authorizedRoles.xpathQuery=*/authorizedRoles/item/role
 
+#######################
+# Cron Job Properties #
+#######################
+# The core size of the job thread pool. After this value any new jobs starting up will have a new thread, and
+# any excess threads will be destroyed after the keep alive time
+crafter.engine.jobs.threadPool.coreSize=10
```

Comment:
Let's discuss.

Reply:
Discussed this and agreed it was ok, just one deployment events watcher job is run for all sites.